ca_driver/src/create_driver.cpp

```diff
@@ -49,6 +49,12 @@ CreateDriver::CreateDriver(ros::NodeHandle& nh) : nh_(nh), priv_nh_("~")
   tf_odom_.child_frame_id = str_base_footprint;
   odom_msg_.header.frame_id = "odom";
   odom_msg_.child_frame_id = str_base_footprint;
+  joint_state_msg_.name.resize(2);
```

Comment:
`joint_state_publisher` [doesn't do it](https://github.com/ros/robot_model/blob/kinetic-devel/joint_state_publisher/joint_state_publisher/joint_state_publisher#L173), so I don't think it's necessary.